src/dbc.cpp

```diff
@@ -28,7 +29,7 @@ const auto whiteSpace = "\\s";
 
 namespace libdbc {
 
-	DbcParser::DbcParser() : version(""), nodes(),
+    DbcParser::DbcParser(bool sortSignals) : version(""), nodes(),

```

Comment:
This should probably be an operation on the class. So you can call the `sort_signals()` method. Not a fan of the control switch how the class starts. I would assume the order when parsing to be the list i get back. 

Also when you say sort how are they sorted by?

Reply:
It was in prepareMessage()
std::sort(m_signals.begin(), m_signals.end()); -> it uses the startbit to determine which one is first